### PR TITLE
Adding russian version to autohiss

### PR DESCRIPTION
**Pull request by Kush1Push1**
# About The Pull Request

Lizardmen and fly persons can speak russian and use their unique tongue's text formation.
I did so for letters "с" (s),"ж" (j) and "з" (z).
For lizards, it extends "c" and replaces "ж" with "ш" (sh).
For fly persons, it extends "з" and replaces "с" with "з".
Doesn't changes original version.

## Why It's Good For The Game

Russian server(s) with TG skyrat codebase will have tongues feature in working state.

## Changelog
:cl:
add: Russian players can use unique races text formation (from lizardmen and fly persons tongues) with their language.
/:cl:

### CODE REVIEW

File: code/modules/surgery/organs/tongue.dm
Comment (GoldenAlpharex) [inline code/modules/surgery/organs/tongue.dm:235]: Just making sure, `З` is different from `3`, right? I don't speak Russian, but I wanted to be certain before hitting the merge button.

**Review by GoldenAlpharex — APPROVED** [review]
Honestly, that's cool, all the more power to you Russians.

File: code/modules/surgery/organs/tongue.dm
Comment (ghost) [reply code/modules/surgery/organs/tongue.dm:235]: But right here - no clue, yeah. And kinda no real difference?

Comment (GoldenAlpharex) [reply code/modules/surgery/organs/tongue.dm:235]: Got it, good to merge then.

**Comment by Kush1Push1** [comment]
To speak the truth, that was really unexpected
Thank you!

**Comment by GoldenAlpharex** [comment]
> To speak the truth, that was really unexpected
> Thank you!

Actually, in hindsight, let's try to get this merged on /tg/, and see if they wouldn't be interested, which I'm sure they could be.

**Comment by GoldenAlpharex** [comment]
> > To speak the truth, that was really unexpected
> > Thank you!
> 
> Actually, in hindsight, let's try to get this merged on /tg/, and see if they wouldn't be interested, which I'm sure they could be.

Never mind I saw the PR upstream.

**Pull request merged**
